datasets/ethics/README.md

```diff
@@ -0,0 +1,210 @@
+---
+annotations_creators:
+- crowdsourced
+- expert-generated
+language_creators:
+- crowdsourced
+- expert-generated
+languages:
+- en-US
+licenses:
+- mit
+multilinguality:
+- monolingual
+pretty_name: ETHICS
+size_categories:
+- unknown
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- acceptability-classification
+- intent-classification
+---
+
+# Dataset Card for ETHICS
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Fields and Instances](#data-instances)
```

Comment:
We try to use the same dataset card structure for all the cards, could you re-add this please ?
```suggestion
  - [Data Instances](#data-instances)
  - [Data Fields](#data-fields)
```

The `Data Fields` section can just list the name of each field (scenario, excuse, etc.) and give a short description + say the type (integer, string)